sdk/appconfiguration/app-configuration/test/public/index.spec.ts

```diff
@@ -1140,4 +1155,308 @@ describe("AppConfigurationClient", () => {
       });
     });
   });
+
+  describe("FeatureFlag configuration setting", () => {
```

Comment:
Feel free to break these tests out into their own .spec.ts file, btw. The index.ts was there when I started and I always thought it'd be nice to have them be in separate files but never got a chance.

Reply:
Split!